vita3k/emuenv/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_library(
+	emuenv
+	INTERFACE
+	include/emuenv/app_util.h
+	include/emuenv/state.h
+	include/emuenv/window.h
+)
+
+target_include_directories(emuenv INTERFACE include ${PSVPFSPARSER_INCLUDE_DIR})
```

Comment:
Btw (that was the cas before this pr, I'm just thinking about it now), why do the psvpsfparser headers must be included with emuenv? It does not use them and even if it were the case they should be linked, not included.

Reply:
I've checked it and they can be removed so I've done so.